oshi-core/src/test/java/oshi/util/UtilTest.java

```diff
@@ -42,5 +42,11 @@ public void testSleep() {
         then = now - 550;
         Util.sleepAfter(then, 500);
         assertTrue(System.currentTimeMillis() - now < 500);
+
+        String LicenseKeytest  = Util.generateLicenseKey();
+        System.out.println(LicenseKeytest);
```

Comment:
Rather than printing it out, use an `assert*()` test method.  Simplest might be asserting the string is not null; you could possibly `String.split()` on the delimiter and assert that the resulting array has a length of at least 4.